app/_kong_plugins/degraphql/examples/enable-degraphql.yaml

```
@@ -1,14 +1,13 @@
-description: |
+description: Enable the DeGraphQL plugin to disable regular Gateway Service function. 
+
+extended_description: |
   Enable the DeGraphQL plugin to disable regular Gateway Service function. 
   
-  Once the plugin is enabled, it builds the path and GraphQL query to hit the GraphQL service with. 
-  Any connected Gateway Services represent your REST API and not the GraphQL endpoint itself. 
+  Once the plugin is enabled, it builds the path and GraphQL query to connect to the GraphQL service with. 
+  Connected Gateway Services represent a REST API and not the GraphQL endpoint itself. 
   {{site.base_gateway}} will return a `404 Not Found` status code if no DeGraphQL routes have been configured.
-
   For a complete tutorial which shows you how to create a service and GraphQL routes, see [Map URIs into GraphQL queries with DeGraphQL](/how-to/map-uris-into-graphql-queries/).
 
```

```suggestion
title: 'Enable DeGraphQL'
```
The missing title is breaking the build.